src/pipeline.ml

```diff
@@ -509,47 +524,41 @@ module Ocaml_org = struct
     in
     pipelines, additional_build_args
 
-  let extras () =
-    let tarsnap =
-      let monthly = Current_cache.Schedule.v ~valid_for:(Duration.of_day 30) () in
-      Current_ssh.run ~schedule:monthly "watch.ocaml.org" ~key:"tarsnap" (Current.return ["./tarsnap-backup.sh"])
-    in
-    let peertube =
-      let weekly = Current_cache.Schedule.v ~valid_for:(Duration.of_day 7) () in
-      let image = Cluster.Watch_docker.pull ~schedule:weekly "chocobozzz/peertube:production-bookworm" in
-      Cluster.Watch_docker.service ~name:"infra_peertube" ~image ()
-    in
-    [tarsnap; peertube]
+  module Watch_docker = Current_docker.Make(struct let docker_context = Some "watch.ocaml.org" end)
```

Comment:
This is moved in from the Cluster module, where it was defined alongside the other functor applications, even tho it wasn't used in that context.